docs/packages/markdown/parseMarkdown.js

```diff
@@ -6,7 +6,7 @@ const prism = require('./prism');
 const headerRegExp = /---[\r\n]([\s\S]*)[\r\n]---/;
 const titleRegExp = /# (.*)[\r\n]/;
 const descriptionRegExp = /<p class="description">(.*?)<\/p>/s;
-const headerKeyValueRegExp = /(.*?): (.*)/g;
+const headerKeyValueRegExp = /(.*?):[\r\n]?\s+(.*)/g;
```

Comment:
This is worth a new test case, this logic is quite brittle.

Reply:
I'll open another PR only to fix it. This PR became more than a blog post.

Reply:
For context, I faced the same problem with the v5 core blog post, I fixed it by not mentioning every one of the team 😆.  In my case prettier was putting one author per line

Reply:
This fix with a test case is in https://github.com/mui-org/material-ui/pull/29633.